resources/product-listing.js

```diff
@@ -20,11 +20,35 @@ function ProductListing(shopify) {
 }
 
 assign(ProductListing.prototype, omit(base, [
-  'create',
-  'delete',
   'update'
 ]));
 
+/**
+ * Adds the productId to a sales channel
+ *
+ * @param {Number} productId ProductID to add to the sales channel
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+ProductListing.prototype.create = function addProductId(productId) {
+  const url = this.buildUrl(`${productId}`);
+  return this.shopify.request(url, 'PUT', undefined, {})
```

Comment:
The [example](https://help.shopify.com/api/reference/productlisting#create) does not use an empty body. Can we do something like this?

```js
/**
 * Creates a product listing.
 *
 * @param {Number} productId The ID of the product to publish
 * @param {Object} [params] Body parameters
 * @return {Promise} Promise that resolves with the result
 * @public
 */
ProductListing.prototype.create = function create(productId, params) {
  params || (params = { product_id: productId });
  const url = this.buildUrl(productId);
  return this.shopify.request(url, 'PUT', this.key, params || { product_id: productId });
};
```